SYCL/ESIMD/api/functional/ctors/ctor_copy.cpp

```diff
@@ -23,7 +23,6 @@
 
 #include "common.hpp"
 
-using namespace sycl;
 using namespace sycl::ext::intel::experimental::esimd;
 using namespace esimd_test::api::functional::ctors;
 using namespace esimd_test::api::functional;
```

Comment:
```suggestion
using namespace esimd_test::api::functional;
```
Let's be consistent with the `ctors_default.cpp`

Reply:
I've done it in this commit: 77dd9fb4a5fdb0e2058cd42506413458ce4ec498